components/brave_rewards/browser/android_util.cc

```diff
@@ -5,13 +5,15 @@
 
 #include "base/json/json_reader.h"
 #include "brave/components/brave_rewards/browser/android_util.h"
+#include "brave/common/brave_channel_info.h"
```

Comment:
sorting

Reply:
Raised https://github.com/brave/brave-browser/issues/9166 to resolve across Brave Ads